src/qt/bitcoingui.cpp

```diff
@@ -697,7 +697,7 @@ void BitcoinGUI::setNumConnections(int count)
     labelConnectionsIcon->setToolTip(tr("%n active connection(s) to Bitcoin network", "", count));
 }
 
-void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, double nVerificationProgress)
+void BitcoinGUI::setNumBlocks(int count, const QDateTime& blockDate, double nVerificationProgress, bool header)
```

Comment:
Please avoid adding bool parameter to function. This is completely intelligible from the caller standpoing.

You can either use a typed enum or create 2 function using the same private utility method for the common code.